src/flask_session/sessions.py

```diff
@@ -13,11 +13,7 @@
 from itsdangerous import Signer, BadSignature, want_bytes
 
 
-PY2 = sys.version_info[0] == 2
-if not PY2:
-    text_type = str
-else:
-    text_type = unicode
+text_type = str
```

Comment:
This should be removed altogether and `str` used directly.

Reply:
Addressed in the latest commit